<?xml version="1.0"?>
<launch>
    <arg name="name" default="ros_env" doc="Set the physics bridge name."/>
    <group ns="$(arg name)">
        <node name="physics_bridge" pkg="eager_bridge_gazebo" type="gazebo_node.py" output="screen"/>
    </group>
</launch>
